android/android.gradle

```diff
@@ -5,8 +5,10 @@ android {
     defaultConfig {
         minSdkVersion CONFIG.versions.android.sdk.min
         targetSdkVersion CONFIG.versions.android.sdk.target
-        versionName project.MODULE_VERSION
-        versionCode Integer.parseInt(MODULE_VERSION_CODE)
+        versionName project.version
+        if(project.hasProperty('versionCode')) {
```

Comment:
Could we maybe decompose `versionName` and automatically apply the formula that's been applied manually all this time (`<M>.<m>.<p>[-SNAPSHOT]` → `(M * 10_000 + m * 100 + p)`) ?

Reply:
tbh that's something you can do in a project if it really needs the version code - I don't (actively) use it anywhere, do you?